src/app/components/Proposal/Proposal.tsx

```diff
@@ -20,26 +18,23 @@ export default function Proposal({ content }: ProposalProps): JSX.Element {
 
   const voteTypes = ['pro', 'neutral', 'remarks', 'concerns']
   const voteTypeSymbols = {
-    pro: mdiPartyPopper,
-    neutral: mdiCheck,
-    remarks: mdiHelpRhombusOutline,
-    concerns: mdiAlertRhombusOutline,
+    pro: <FaStar size="1em" />,
+    neutral: <FaCheck size="1em" />,
+    remarks: <FaRegQuestionCircle size="1em" />,
+    concerns: <FaExclamationCircle size="1em" />,
```

Comment:
for other sizes you use px, 
poople told me its better to stick with one ` Einheit`